API_Kraken/get_bitcon_price.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import datetime
+
+get_request = requests.get('https://api.kraken.com/0/public/Ticker?pair=XBTUSD')
+price = get_request.json()['result']['XXBTZUSD']['a'][0]
+date = datetime.datetime.now().strftime("%c")
+value = ('Price of bitcon for: ' + date + ' is: ' + price + ' USD')
+try:
+    print(value)
+except:
+    print("An error has occured")
+
+value_txt = {"Price of bitcon for:" : date, "is" : price}
+
+file = open("bitcoin_price.txt", "w")
```

Comment:
If we run script 2 times then last execution will override first one. If we open file in append mode:

`file = open("bitcoin_price.txt", "a")`

Then we will incrementally add new line to the file which we wanted to achive